tools/generator-go-sdk/internal/generator/templater_id_parser.go

```diff
@@ -83,6 +85,16 @@ type %[1]s struct {
 	return &out, nil
 }
 
+func (r resourceIdTemplater) registerId() string {
+	wordifiedName := wordifyString(r.name)
+	return fmt.Sprintf(`
+	// Adds %[1]s to a register of known ids
```

Comment:
could we remove this, else this'll be a linter warning:

```suggestion
```

(Due to the placement of this comment, linters/IDEs will highlight that the first word should be the name of the function since comments above the function are used as a string in the documentation for the package)